docs/source/nlp/nemo_megatron/retro/retro_model.rst

```diff
@@ -1,469 +1,170 @@
-| model.init_method_std            | 0.02        | standard deviation of the zero mean normal distribution used for weight initialization |
+| model.init_method_std            | 0.023       | standard deviation of the zero mean normal distribution used for weight initialization |
 +----------------------------------+-------------+----------------------------------------------------------------------------------------+
 | model.hidden_dropout             | 0.1         | dropout probability for hidden state transformer                                       |
 +----------------------------------+-------------+----------------------------------------------------------------------------------------+
 | model.attention_dropout          | 0.1         | dropout probability in the attention layer                                             |
 +----------------------------------+-------------+----------------------------------------------------------------------------------------+
-| model.ffn_dropout                | 0           | dropout probability in the feed-forward layer                                          |
+| model.ffn_dropout                | 0.1         | dropout probability in the feed-forward layer                                          |
 +----------------------------------+-------------+----------------------------------------------------------------------------------------+
 
-
-Option 1: Train the NeMo RETRO model *without* mu-Transfer
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-An example RETRO pre-training script is:
-
-.. code-block:: bash
-
-    python examples/nlp/language_modeling/megatron_retro_pretraining.py \
-        trainer.devices=8 \
-        trainer.num_nodes=2 \
-        trainer.accelerator=gpu \
-        trainer.max_steps=800000 \
-        trainer.precision=16 \
-        exp_manager.exp_dir=/result/retro_model \
-        model.apply_query_key_layer_scaling=False \
-        model.tensor_model_parallel_size=8 \
-        model.optim.name=adamw \
-        model.enc_num_layers=2 \
-        model.dec_num_layers=32 \
-        model.enc_cross_attention=[0] \
-        model.dec_cross_attention=[8,11,14,17,20,23,26,29,31] \
-        model.hidden_size=4096 \
-        model.ffn_hidden_size=16384 \
-        model.num_attention_heads=32 \
-        model.tokenizer.merge_file=/dataset/gpt2-merges.txt \
-        model.tokenizer.vocab_file=/dataset/gpt2-vocab.json \
-        model.data.data_prefix=[/result/pubmed_eval_text_document] \
-        model.data.knn_index=[dataset/pubmed_knn_final.save] \
-        model.data.retrieval_prefix=/result/pubmed_eval_text_document \
-        model.micro_batch_size=8
-
-During the training, launch Tensorboard to monitor training like so:
+Below is an example RETRO pre-training script. The rest of the arguments values are retrieved from ``examples/nlp/language_modeling/conf/megatron_retro_config.yaml``.
```

Comment:
Below is an example RETRO pre-training script. The rest of the arguments are retrieved from ``examples/nlp/language_modeling/conf/megatron_retro_config.yaml``.